### PR TITLE
Upgrading to os-maven-plugin to 1.4.1.Final

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

**Comment by nmittler** [comment]
@zhangkun83 PTAL

File: build.gradle [hunk 1]
```diff
@@ -8,12 +8,12 @@ description = 'A Gradle plugin that detects the OS name and architecture, provid
 group = 'com.google.gradle'
 
 // The major and minor versions are aligned with the Maven plugin's.
-version = '1.4.0'
+version = '1.4.1'
```
Comment (zhangkun83) [inline build.gradle]: The version should be `1.4.1-SNAPSHOT`. I should've made the change right after the `1.4.0` release.

Comment (nmittler) [inline build.gradle]: done.

**Comment by nmittler** [comment]
PTAL

**Comment by zhangkun83** [comment]
Squashed and merged 995427255a77072d38ed28c8eee4b02ff48a47f9

**Comment by nmittler** [comment]
Sweet, thanks!

**Pull request closed without merging**